src/renderer/dx/DxRenderer.cpp

```diff
@@ -659,7 +801,15 @@ void DxEngine::_ReleaseDeviceResources() noexcept
     {
         _haveDeviceResources = false;
 
+        // Destroy Terminal Effect resources
+        _renderTargetView.Reset();

```

Comment:
Seems like a mistake that these weren't freed. I checked `operator&` and it correctly frees the resource so doesn't seem there were leaks due to this.

Reply:
Woops. Good catch.